### PR TITLE
Updated rails warning

### PR DESCRIPTION
**Pull request by stanleyguevara**

### CODE REVIEW

File: android.md [hunk 1]
```diff
@@ -12,13 +12,18 @@ additionalNavigation : [
...
+<p>As a result, a new account must be set up on the Sensorberg IoT platform in order to continue your projects.</p>
+<p>Simply create an account by visiting <a href="http://portal.sensorberg.com" target="_blank">portal.sensorberg.com</a>. You have the option to manually move your data (e.g., beacons, campaigns etc) to the new platform and continue to manage your account.&nbsp;</p>
+<p>Alternatively, you may contact our support team at <a href="mailto:support@sensorberg.com">support@sensorberg.com</a>&nbsp;for assistance in the migration procedure.</p>
+<p>Once you have created the new account on Sensorberg IoT platform, you may contact Support to <a href="https://support.sensorberg.com/hc/en-us/articles/211475789-How-can-I-delete-my-account-" target="_blank">delete your old account</a>.</p>
+<p>For more useful knowledge articles, please visit our&nbsp;<a href="https://support.sensorberg.com/hc/en-us/categories/115000135909-Sensorberg-IoT-Enterprise-Platform" target="_blank">Knowledge Center.</a></p>
+<p>The latest current artifact we recommend to use is:</p>
```
Comment (falkorichter) [inline android.md]: We don´t recommend. The latest artifact:

**Comment by falkorichter** [comment]
@tagyro why don´t we have similar changes on the iOS page?

File: android.md [hunk 2]
```diff
@@ -12,13 +12,18 @@ additionalNavigation : [
...
 ---
 
-<div class="callout callout-alert">
-    <h1><i class="fa fa-exclamation-triangle"></i>Note about the new portal:</h1>
-    <p>To use portal.sensorberg.com you must use a different artifact of the SDK</p>
-{% highlight groovy %}
-compile 'com.sensorberg.sdk:android-sdk:2.3.2-RAILS'
+<div>
+<h2>Sensorberg Android SDK v2.2.0 Migration to New Platform</h2>
+<p>The Sensorberg Android SDK v2.2.0 will now be using our newly implemented Sensorberg IoT platform, which is implemented on a RAILS Back-end. This new system upgrade will ensure better functionality, project use cases and&nbsp;simpler user experience.</p>
```
Comment (falkorichter) [inline android.md]: 2.3.2? Why 2.2.0???

File: android.md [hunk 3]
```diff
@@ -12,13 +12,18 @@ additionalNavigation : [
...
+<p>As a result, a new account must be set up on the Sensorberg IoT platform in order to continue your projects.</p>
+<p>Simply create an account by visiting <a href="http://portal.sensorberg.com" target="_blank">portal.sensorberg.com</a>. You have the option to manually move your data (e.g., beacons, campaigns etc) to the new platform and continue to manage your account.&nbsp;</p>
+<p>Alternatively, you may contact our support team at <a href="mailto:support@sensorberg.com">support@sensorberg.com</a>&nbsp;for assistance in the migration procedure.</p>
+<p>Once you have created the new account on Sensorberg IoT platform, you may contact Support to <a href="https://support.sensorberg.com/hc/en-us/articles/211475789-How-can-I-delete-my-account-" target="_blank">delete your old account</a>.</p>
```
Comment (falkorichter) [inline android.md]: No need to tell them how to delete accounts...

File: android.md [hunk 4]
```diff
@@ -12,13 +12,20 @@ additionalNavigation : [
 ]
 ---
 
-<div class="callout callout-alert">
-    <h1><i class="fa fa-exclamation-triangle"></i>Note about the new portal:</h1>
-    <p>To use portal.sensorberg.com you must use a different artifact of the SDK</p>
+<div>
+<h2>SDK v2.2.0-RAILS Migration to New Platform</h2>
+<p>The SDK v2.2.0-RAILS (and higher) is using our newly implemented Sensorberg IoT RAILS platform.
```
Comment (falkorichter) [inline android.md]: why 2.2.0?

File: android.md [hunk 5]
```diff
@@ -12,13 +12,20 @@ additionalNavigation : [
...
+This new system upgrade will ensure better functionality, project use cases and&nbsp;simpler user experience.</p>
+<p>As a result, a new account must be set up on the Sensorberg IoT platform in order to continue your projects.
+This applies to every dependency change from <b>non-RAILS</b> version to <b>-RAILS</b> based version.</p>
+<p>Simply create an account by visiting <a href="http://portal.sensorberg.com" target="_blank">portal.sensorberg.com</a>. You have the option to manually move your data (e.g., beacons, campaigns etc) to the new platform and continue to manage your account.&nbsp;</p>
+<p>Alternatively, you may contact our support team at <a href="mailto:support@sensorberg.com">support@sensorberg.com</a>&nbsp;for assistance in the migration procedure.</p>
+<p>Once you have created the new account on Sensorberg IoT platform, you may contact Support to <a href="https://support.sensorberg.com/hc/en-us/articles/211475789-How-can-I-delete-my-account-" target="_blank">delete your old account</a>.</p>
```
Comment (falkorichter) [inline android.md]: delete this paragraph. no need to delete an old account.

Comment (falkorichter) [inline android.md]: we don´t need to awake a sleeping lion

File: android.md (hunk 4 above)
Comment (stanleyguevara) [inline android.md]: Beacuse it was firs -RAILS based version we released:
http://jcenter.bintray.com/com/sensorberg/sdk/android-sdk/

File: android.md (hunk 5 above)
Comment (stanleyguevara) [inline android.md]: If that's all I'll rebase, force-update to make this PR one commit and we're good to go?

File: android.md (hunk 4 above)
Comment (falkorichter) [inline android.md]: We can´t move backward. Maybe even call 2.4.0

I´m almost done with my meeting, let´s hangout later.

Comment (stanleyguevara) [inline android.md]: We also can't rewrite history, and the fact is there was 2.2.0 and 2.2.0-RAILS,
then only -RAILS versions, so this is breaking point we need to mention somehow.

I'd make the header "SDK migration to RAILS platform" and "Starting with SDK 2.2.0-RAILS we're using newly implemented blah blah blah..."

**Comment by tagyro** [comment]
@falkorichter mostly because on iOS it's pretty well explained already - I'll have a look if/how to make it clearer

**Pull request merged**